NNPTPZ1/ComplexNumber.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+
+namespace NNPTPZ1.Mathematics {
+    public class ComplexNumber {
+        public double RealPart { get; set; }
+        public float ImaginaryPart { get; set; }
+
+        public static readonly ComplexNumber Zero = new ComplexNumber() {
+            RealPart = 0,
+            ImaginaryPart = 0
+        };
+
+        public override bool Equals(object obj) {
+            if (obj is ComplexNumber) {
+                ComplexNumber complexNumber = obj as ComplexNumber;
+                return complexNumber.RealPart == RealPart && complexNumber.ImaginaryPart == ImaginaryPart;
+            }
+            return base.Equals(obj);
+        }
+
+        public ComplexNumber Multiply(ComplexNumber multiplicand) {
```

Comment:
please place binary math operations methods to one place; now they are mixed with getters and other methods